README.md

```diff
@@ -20,16 +20,125 @@
     </a>
 </div>
 
-## DISCLAIMER: STILL IN DEVELOPMENT
+## Overview
 
-This project is still under heavy development. Feel free to contact us on [Discord](https://discord.snapshot.org)!
+Governance mechanisms obey a trilemma between **decentralisation**, **cost**, and **flexibility**. On the one end you have systems like [Snapshot](https://snapshot.org/#/) that provide a very cheap experience [1] with lots of flexibility. However it relies on one trusting the Snapshot off-chain protocol to deliver the verdict of a particular governance proposal and to not censor votes. Flexibility comes from the wide range of voting strategies than one can employ to calculate the voting power for each user. 

```

Comment:
I believe it's "On one hand" / "On the other hand" ?

Reply:
https://dictionary.cambridge.org/dictionary/english/on-the-one-hand-on-the-other-hand

Reply:
yeah, sounds better